modules/c/partials/release-notes/couchbase-mobile-c-release-note.3.2.0.adoc

```diff
@@ -1,25 +1,42 @@
 [#maint-3-2-0]
-== 3.2.0 -- TODO 2024
+== 3.2.0 Beta 1 -- March 2024
```

Comment:
I think it sounds like we should NOT have this section, and instead at most a "IMPORTANT: 3.2.0 is not released for this platform" banner instead?

Reply:
IMPORTANT admonition is a good shout